app/templates/search.html

```diff
@@ -1,14 +1,39 @@
 {% extends "base.html" %}
 {% import '_snippets.html' as snippets %}
+{% block scripts %}
+  <script>
```

Comment:
@subhankarb no i didn't say that ;-)

This is an unfortunate example of a miscommunication. To try and avoid this I specifically wrote out the list of things to do yesterday in a comment here:

https://github.com/frictionlessdata/dpr-api/issues/200#issuecomment-278276165

There is no mention of hiding the search box in the navbar :wink: